tests/piqueserver/test_badmin.py

```diff
@@ -4,6 +4,7 @@
 
 
 class TestBadmin(unittest.TestCase):
+    # Requires time to be positive
     def test_score_grief1(self):
```

Comment:
You should make the name of the test describe it's purpose. For example in this case, `test_score_time_is_positive`. When the test fails, the name of the test will appear. If it has a name that includes numbers, it is not clear which test exactly failed without looking it up in the source file.